### DISCUSSION TITLE
How may I update to version 3.9.2 onrailway?

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3053] [Q&A] [answered]
Hi, I'm using railway to host my code-server and it's been working really well. Is there a way to update to the latest version?

**Comment by jsjoeio** [comment, marked as answer]
Great question! @bpmct answered that [here](https://github.com/cdr/code-server/discussions/2902#discussioncomment-491369)

```Dockerfile
FROM codercom/code-server:3.9.2 
```

Though, be warned, there are known issues with 3.9.2 that we're actively fixing. See [here](https://github.com/cdr/code-server/discussions/3038)
